tests/integration/conftest.py

```diff
@@ -125,8 +125,8 @@ def load_wazuh_basic_configuration():
 @pytest.fixture()
 def set_wazuh_configuration(test_configuration: dict) -> None:
     """Set wazuh configuration
-
     Args:
+    
```

Comment:
You applied the change in the wrong line. Please, don't modify this file to avoid conflicts wit other PRs.